CriticalMass/ThemeController.swift

```diff
@@ -69,7 +69,6 @@ class ThemeController {
         // Custom Views
         SettingsFooterView.appearance().versionTextColor = theme.titleTextColor
         SettingsFooterView.appearance().buildTextColor = theme.titleTextColor
-        SettingsGithubTableViewCellTableViewCell.appearance().arrowTintColor = .settingsOpenSourceForeground
```

Comment:
This line made problems after adding the SnapshotTest and it is not really needed here, because the `arrowTintColor` is not set dependent on the theme.